.nvmrc

```diff
@@ -0,0 +1 @@
+v12.18.0
```

Comment:
[Node Version Manager](https://github.com/nvm-sh/nvm) has pretty widespread usage and ensures we're all running the same version of Node.js for local development.

I'm guessing this also explains the large `package-lock.json` diff, which is probably less than ideal 😬. Let me know if you have thoughts on how to ease this review...

Reply:
FYI I reverted the `jest` upgrade for now to minimize the diff.